### DISCUSSION TITLE
Favorite publications with "bubble up"

### DISCUSSION THREAD

**Discussion by randy** [dailydotdev/daily#1095] [Feature Requests]
I'd really love to see the option to "favorite" certain contributors. Their posts would then "bubble up" to the top of my feed in some manner. For example, I like reading DHH's posts to Hey World, but I rarely catch them in my feed even though they're often tagged with topics I follow. 

Instead, I have shortcut set... but if I follow the shortcut and notice a new post, reading it doesn't seem to count toward the Weekly reading goal. So then I have to copy the title, search, click a couple of times, then read. Since I understand that shortcuts are not internal to daily.dev, I don't expect reading that material to count, so to me, the only viable solution is this feature request.

How the "bubble-up" part works, I don't know... but I have faith in the daily.dev team! 😁

**Comment by idoshamun** [comment]
Hi @randy,

Thanks for the faith in us 😬
We will soon roll out a "subscribe to source" feature that will notify you on every new post of a source.
You can also add our companion widget to be able to track the daily.dev post of every web page you visit. It requires extra permissions but we never track your reading history.

Sidenote: weekly goal is about to get revamped 😈
